src/pods/canvas/components/table/components/database-table-row.component.tsx

```diff
@@ -31,7 +31,8 @@ const COLUMN_NAME_PIXEL_WIDTH = calculateColumNameWidth(
   TABLE_CLEAN_WIDTH
 );
 
-const SPACE_ARROW_TEXT = TABLE_CONST.FONT_SIZE * 1.5;
+const SPACE_ARROW_TEXT = TABLE_CONST.FONT_SIZE * 1.35;
+const PIXELS_CENTER_ARROW = 1.5;
```

Comment:
I guess this tweakings are not related with Zoom issue? If that's the case for the next time, create separate issue and branch for this, right now is adding noise to the PR (if not please could you indicate how is related to this zoom factor)?

Reply:
Sorry, I noticed that mistake while playing with the zoom, next time I will create a new issue.